CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
```

Comment:
The intent of the phrase `sexualized language` clearly is not meant to include the name of the project, but for clarity's sake it might make sense to explicitly permit discussion which specifically refer to the project itself without any innuendo.  I don't have a great suggestion for specific verbiage, however.  And maybe people are smart enough to understand intent.  But in the linked issue, at least one commenter is clearly quite confused about language -- so I guess never underestimate how clueless people can be. :/

Reply:
> The intent of the phrase `sexualized language` clearly is not meant to include the name of the project, but for clarity's sake it might make sense to explicitly permit discussion which specifically refers to the project itself without any innuendo. I don't have a great suggestion for specific verbiage, however. And maybe people are smart enough to understand intent. But in the linked issue, at least one commenter is clearly quite confused about language -- so I guess never underestimate how clueless people can be. :/

Then should I remove the statement?

Reply:
I don't personally have a strong opinion. Simply removing "language or" might be ok, or might not. I'd rather hear input from community leaders and not make that decision in a vacuum. :)